src/Components/Calendar.css

```diff
@@ -13,14 +13,18 @@
   font-family: Chalkduster, sans-serif;
 }
 
+.calendarDay {
+  height: 100%; /* makes the box equal to 100% of its parent's height */
+  min-height: 50vh; /* 50% of viewport height */
+}
+
 .box1 {
   color: black;
   background-color: #a0df76;
```

Comment:
since you created a second clss in the css I would suggest moving out the common item from box1, box2, ... into it as well like the border, margin and shadow.

Reply:
good suggestion...done!